src/Simplify.elm

```diff
@@ -300,7 +300,7 @@ Destructuring using case expressions
     --> x
 
 
-### Maybe
+### Maybes
```

Comment:
Good catch :+1:

Reply:
Same up there with "field access" header. Tho I don't have a good plural for "accesses"

Reply:
I think "accesses" is the correct plural. But "Field access" is fine also. Leave that one as is :man_shrugging: